test/k6/src/tests/orders_email.js

```diff
@@ -64,169 +61,119 @@ export function setup() {
   return data;
 }
 
-// 01 - POST email notification order request
 function TC01_PostEmailNotificationOrderRequest(data) {
-  var response, success;
+  let success;
 
-  response = ordersApi.postEmailNotificationOrder(
+  const response = ordersApi.postEmailNotificationOrder(
     JSON.stringify(data.emailOrderRequest),
     data.token
   );
-  var selfLink = response.headers["Location"];
 
   success = check(response, {
-    "POST email notification order request. Status is 202 Accepted": (r) =>
-      r.status === 202,
+    "POST email notification order request. Status is 202 Accepted": 
+      (r) => r.status === 202
+  });
+
+  stopIterationOnFail("POST email notification order request failed", success);
+
+  success = check(response, {
     "POST email notification order request. Location header providedStatus is 202 Accepted":
-      (r) => selfLink,
+      (r) => r.headers["Location"],
     "POST email notification order request. Recipient lookup was successful":
       (r) => JSON.parse(r.body).recipientLookup.status == 'Success'
   });
 
-  addErrorCount(success);
-
-  if (!success) {
-    stopIterationOnFail(success);
-  }
-
-  return selfLink;
+  return response.headers["Location"];
```

Comment:
is it explicit that this is the self link for the order? Might be easier to follow understand the return value if it is named despite it not being used further?

Reply:
Reverted the change.